src/test/regress/expected/multi_foreign_key.out

```diff
@@ -414,20 +414,20 @@ ABORT;
 -- test foreign constraint creation
 -- test foreign constraint creation with not supported parameters
 ALTER TABLE referencing_table ADD CONSTRAINT test_constraint FOREIGN KEY(ref_id) REFERENCES referenced_table(id) ON DELETE SET NULL;
-ERROR:  cannot create foreign key constraint
-DETAIL:  SET NULL or SET DEFAULT is not supported in ON DELETE operation.
+ERROR:  cannot create foreign key constraint since relations are not colocated
+DETAIL:  Foreign key constraint can only be created on co-located tables or reference tables.
```

Comment:
Minor: The updated error details seems less useful and we're probably changing the coverage of the tests. Could you maybe change `master_create_distributed_table` to `create_distributed_table` and make sure that we keep giving the same errors? 

I think it is expected ` Foreign key constraint can only be created on co-located tables.	 ` -> `Foreign key constraint can only be created on co-located tables or reference tables` 
But not `SET NULL or SET DEFAULT is not supported in ON DELETE operation.` -> `Foreign key constraint can only be created on co-located tables or reference tables`